packages/advanced-logic/specs/payment-network-erc777-stream-0.1.0.md

```diff
@@ -0,0 +1,445 @@
+# Payment Network - ERC777 Stream
```

Comment:
what is the name of the PN? `ERC777-stream` or `ERC777-fee-superfluid`

File name + title do not match the rest of the file

Reply:
I've started with ERC777-fee-superfluid by adapting from ERC20-fee-proxy but ERC777-stream seems easier and more generic. Now that fees are by default it seems redundant to add it to the name. And for those who don't know yet SuperFluid stream is more expressive thand superfluid.

Reply:
Updated everything to `ERC777-stream` 👍